recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
I can understand the argument about the checksum as it only shows up once, but the name does show up a few places.

Admittedly the name is not likely going to change in the same package, but I can see someone borrowing and tweaking one of our recipes to add some new package here. Wouldn't it be nice if we didn't have to catch every occurrence of the name they forgot to change? :thought_balloon: